### ISSUE TITLE
[124] Issue #124 - December 13, 2018

### ISSUE THREAD

**Issue by BasThomas** [SwiftWeekly/swiftweekly.github.io#429]
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/master/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

**Comment by BasThomas** [comment]
I'm out of office and will not be able to write the next issue. I can review it, so if somebody else would like to take an issue, that would be fantastic!

**Comment by fassko** [comment]
https://forums.swift.org/t/keypaths-tuples-and-swift-5/18465

KeyPaths, Tuples and Swift 5

**Comment by fassko** [comment]
https://forums.swift.org/t/proposal-to-add-cancellation-abilities-for-async-await/18419

Interesting take on Async/Await to add cancellation abilities.

P.S. I really love how community is pushing it forward.

**Comment by fassko** [comment]
https://forums.swift.org/t/rfc-moving-swiftnio-ssl-to-boringssl/18280

RFC: Moving SwiftNIO SSL to BoringSSL

**Comment by fassko** [comment]
https://forums.swift.org/t/review-2-of-se-0237-introduce-withunsafe-mutable-bufferpointerifsupported-methods/18418

Review #2 of SE-0237: Introduce withUnsafe Mutable BufferPointerIfSupported methods

> second review of the revised SE-0237

**Comment by fassko** [comment]
https://forums.swift.org/t/accepted-with-modifications-se-0236-package-manager-platform-deployment-settings/18420

**[Accepted with Modifications] SE-0236: Package Manager Platform Deployment Settings**

> This makes the API .macOS("10.13") instead of .macOS(MacOSVersion("10.13")).

**Comment by fassko** [comment]
Activity in Server working group

https://forums.swift.org/t/november-29th-2018/18400

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
We all know @mattt obsession of Strings 🙃

https://forums.swift.org/t/pitch-unicode-named-character-escape-sequence/18396

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
https://forums.swift.org/t/speeding-up-swiftsyntax-by-using-the-parser-directly/18493

tl;dr; Providing direct access to the parser speeds up SwiftSyntax 8x, and it becomes 2x faster than the legacy sourcekitd syntactic request

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
https://oleb.net/2018/sequence-head-tail/

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
Would be cool to run Swift in Amazon Lambda. Let's see where it goes.

https://forums.swift.org/t/aws-lambda-runtime-api/18498

**Comment by BasThomas** [comment]
I think I haven't included that indeed, so feel free :)

**Comment by BasThomas** [comment]
https://twitter.com/swift_unwrapped/status/1069593158445662209

**Comment by fassko** [comment]
This should be mentioned https://twitter.com/swiftlybrief/status/1069592028701437953

**Comment by fassko** [comment]
https://nshipster.com/homebrew/

**Swift Program Distribution with Homebrew**

> For iOS and macOS frameworks, we use Carthage or CocoaPods. For Swift packages, we use Swift Package Manager. And when it comes time to distribute a command-line tool built in Swift, we use Homebrew.

**Comment by BasThomas** [comment]
>https://nshipster.com/homebrew/ ...

@fassko not sure if this is relevant?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
> > [nshipster.com/homebrew](https://nshipster.com/homebrew/) ...
> 
> @fassko not sure if/how this is relevant to the newsletter?
> 
> Sent with [GitHawk](http://githawk.com)

I'm still thinking if that can make it's place in newsletter. I was thinking that this is other way to distribute Swift programs. SwiftPM isn't the only way.

Your opinion?

**Comment by BasThomas** [comment]
I’d keep it out. I think it’s questionable and there seems to be enough other content as well :) 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
https://twitter.com/mishaldshah/status/1070048389893505024

**Comment by fassko** [comment]
I think this thread is a fun one. https://twitter.com/twostraws/status/1070408833321836545

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
https://forums.swift.org/t/accepted-with-modifications-se-0238-package-manager-target-specific-build-settings/18590

SE-0238 accepted with modifications 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
Result proposal accepted with modifications https://forums.swift.org/t/accepted-with-modifications-se-0235-add-result-to-the-standard-library/18603

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by wongzigii** [comment]
Fun one:

https://twitter.com/modocache/status/979516199498387456

**Comment by fassko** [comment]
https://forums.swift.org/t/accepted-with-modifications-se-0237-introduce-with-contiguous-mutable-storage-if-available-methods/18713

**Comment by BasThomas** [comment]
https://twitter.com/Ilseman/status/1072283524579831808

> Would you like to access the raw UTF-8 code units backing a String? Now you can, thanks to `String.UTF8View.withContiguousStorageIfAvailable` hook:

https://github.com/apple/swift-evolution/blob/master/proposals/0237-contiguous-collection.md

https://github.com/apple/swift/pull/21178

**Comment by BasThomas** [comment]
https://github.com/apple/swift-evolution/pull/966

**Comment by fassko** [comment]
https://github.com/inamiy/SwiftRewriter

https://twitter.com/inamiy/status/1072282778236932097

**Comment by fassko** [comment]
Functional Swift Conference in NYC videos are out https://twitter.com/mbrandonw/status/1072541950954352640

**Comment by fassko** [comment]
https://www.hackingwithswift.com/articles/147/swift-5-is-changing-optional-try

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by wongzigii** [comment]
The official Swift Docker image is now hosted under the Apple organization on GitHub: https://github.com/apple/swift-docker

From https://twitter.com/olebegemann/status/1072816706861625345

**Comment by BasThomas** [comment]
Closed by #431
